packages/react-reconciler/src/__tests__/ReactNewContext-test.js

```diff
@@ -1077,10 +1078,13 @@ describe('ReactNewContext', () => {
 
       // Update
       ReactNoop.render(<App value={2} />);
-      expect(() => expect(Scheduler).toFlushWithoutYielding()).toErrorDev(
-        'calculateChangedBits: Expected the return value to be a 31-bit ' +
-          'integer. Instead received: 4294967295',
-      );
+
+      if (gate(flags => !flags.enableLazyContextPropagation)) {
+        expect(() => expect(Scheduler).toFlushWithoutYielding()).toErrorDev(
+          'calculateChangedBits: Expected the return value to be a 31-bit ' +
+            'integer. Instead received: 4294967295',
+        );
+      }
```

Comment:
I like having else blocks required for in-line feature flags so you either know what the expected behavior is in the other branch or can explain why it's missing, something like:

```suggestion
      } else {
        // TODO: Decide what to do with calculateChangedBits in enableLazyContextPropagation
      }
```